apps/src/lab2/views/components/instructions.module.scss

```diff
@@ -124,25 +122,30 @@
   color: $neutral_dark;
 }
 
+.textContent {
+  height: 100%;
+  overflow-y: auto;
+  padding: 15px 20px 5px 20px;
+  box-sizing: border-box;
+}
+
 %message {
+  position: relative;
   padding: 15px 20px 5px 20px;
   border-radius: 4px;
-  border: dashed $neutral_dark 2px;
   animation: slidein 0.5s;
 }
 
 .message-dark {
   @extend %message;
   background-color: $neutral_light;
   color: $neutral_dark;
-  border: dashed $neutral_light 2px;
 }
 
 .message-light {
   @extend %message;
   background-color: $light_gray_50;
   color: $neutral_dark;
-  border: dashed $light_gray_100 2px;
 }
 
 .buttonInstruction {
```

Comment:
Kind of a tangent, but I deleted a bunch of this styling, not sure why it's showing up here?

Reply:
As discussed, probably just because latest staging isn't merged into this branch.